### DISCUSSION TITLE
elevation of pnemonic78 permissions

### DISCUSSION THREAD

**Discussion by Kaligula0** [GnuCash-Pocket/gnucash-android#143] [Team Posts]
I suggest elevation of pnemonic78 permissions, so that he can also manage (review, merge) PRs efficiently. My time has drastically shrinked last month and I see no hope for me in (at least) a few months perspective 🙄 especially if his rate of PRs doesn't fall down ;)

**Comment by Kaligula0** [comment]
@djbrown @pnemonic78 @gutierri

**Comment by pnemonic78** [comment]
just beware in case i break something

**Reply by gutierri** [reply]
As work progresses, we can use GitHub channels to communicate. If you experience any difficulty with something.

But I've been using the App every day with the changes you made, and I've been happy with the results.

**Comment by Kaligula0** [comment]
Yes, I am aware of this scenario. (I am also aware of the _xz backdoor_ scenario ;p) But while I am the only active reviewer, I am no Java developer. (I contributed to many projects in various languages, also wrote scripts and programs in many languages, but it was always just a little work, mostly small corrections etc based on general understanding of code logic.) I am doing what I can as a reviewer, but I must admit I not always know what the PR does.
Nevertheless I am still interested in this project, and that doesn't change. I just have less time.

**Comment by djbrown** [comment]
I promoted @Kaligula0 and @pnemonic78 as organization owner and repo admin, and demoted @gutierri as member/maintainer.
All changes may be examined by owners in the [audit log](https://github.com/organizations/GnuCash-Pocket/settings/audit-log).
I also updated the [Contributors](https://github.com/GnuCash-Pocket/gnucash-android/wiki/Contributors) wiki page accordingly.

@Kaligula0 if you don't want elevated access, I can demote you back to member/maintainer.

**Comment by Kaligula0** [comment]
We'll see how it goes. I can always ask for demotion. Thanks for looking into it.
